### PR TITLE
Get webhook message endpoint

### PR DESCRIPTION
**Pull request by Xpl0itR**
This is useful for getting the ID of the interaction response message which can be used for adding reactions among other things

### CODE REVIEW

File: restapi.go [hunk 1]
```diff
@@ -2236,6 +2236,23 @@ func (s *Session) WebhookExecute(webhookID, token string, wait bool, data *Webho
 	return
 }
 
+// WebhookMessageGet gets a webhook message.
+// webhookID : The ID of a webhook
+// token     : The auth token for the webhook
+// messageID : The ID of message to get
+func (s *Session) WebhookMessageGet(webhookID, token, messageID string) (message *Message, err error) {
```
Comment (FedorLap2006) [inline restapi.go:2243]: Probably would be better to use here `WebhookMessage` instead.

File: restapi.go [hunk 2]
```diff
@@ -2557,6 +2574,13 @@ func (s *Session) InteractionRespond(interaction *Interaction, resp *Interaction
 	return err
 }
 
+// InteractionResponseGet gets the response to an interaction.
+// appID       : The application ID.
+// interaction : Interaction instance.
+func (s *Session) InteractionResponseGet(appID string, interaction *Interaction) (*Message, error) {
```
Comment (FedorLap2006) [inline restapi.go:2580]: Look at another comment

**Review by FedorLap2006 — COMMENTED** [review]
Probably you should rename this PR to "Get webhook message endpoint". But anyways, I think that's a part of the #951. Also, mind the comments.

**Comment by CarsonHoffman** [comment]
Thank you for the additions!

**Pull request merged**